lib/ansiblelint/__init__.py

```diff
@@ -19,12 +19,12 @@
 # THE SOFTWARE.
 
 from collections import defaultdict
+import codecs
```

Comment:
(future hint) it's probably unnecessary now that we don't have to support Python 2. Out of scope of this PR, of course, but would be a nice improvement separately.